.node-version

```diff
@@ -0,0 +1 @@
+5
```

Comment:
Do you mind adding a trailing new line to this file to follow POSIX standards: https://robots.thoughtbot.com/no-newline-at-end-of-file